syncer/syncer.go

```diff
@@ -28,16 +28,16 @@ import (
 type Syncer struct {
 	client client.Client
 	ctx    context.Context
-	logger logr.Logger
+	logger *logr.Logger
```

Comment:
How is this change necessary for adding toolkit and not a rewrite?

Reply:
Because this comes from the adapter and we are following how the new adapter should be defined.